larray/core/session.py

```diff
@@ -154,6 +200,19 @@ def load(self, fname, names=None, engine='auto', display=False, **kwargs):
         display : bool, optional
             whether or not to display which file is being worked on. Defaults
             to False.
+            
+        Examples
+        --------
+        In one module 
+        
+        >>> arr1, arr2, arr3 = ndtest((2, 2)), ndtest(4), ndtest((3, 2))  # doctest: +SKIP
+        >>> Session([('arr1', arr1), ('arr2', arr2), ('arr3', arr3)]).save('input.h5')     # doctest: +SKIP
+        
+        In another module
+        
+        >>> s = Session()                                 # doctest: +SKIP
+        >>> s.load('input.h5', ['arr1', 'arr2', 'arr3'])  # doctest: +SKIP
+        >>> arr1, arr2, arr3 = s.values()                 # doctest: +SKIP
```

Comment:
I am a bit nervous about recommending unpacking from .values(), because it is not always obvious the order the arrays are in. I would either:
* add a comment warning users the order of variables must be the same than in the session
OR
* use >>> arr1, arr2, arr3 = s['arr1', 'arr2', 'arr3'] instead